internal/daemon/hostdaemon.go

```diff
@@ -175,6 +176,7 @@ func (d *HostDaemon) cniCmdDelHandler(req *cnitypes.PodRequest) (*cni100.Result,
 }
 
 func (d *HostDaemon) Listen() (net.Listener, error) {
+	d.wg.Add(1)
```

Comment:
Shouldn't we call wg.Add right before we create the go routine?

Reply:
Which go-routine?